src/app/+dashboard/home/home.component.html

```diff
@@ -70,17 +72,22 @@
 <ng-template #homeSearchBox>
   <div class="row justify-content-center">
     <div class="col-lg-6 col-md-8 col-sm-10 col-xs-12">
-      <scholars-search-box [styles]="getSearchStyles() | async" [view]="discoveryView | async" placeholder="{{ 'SHARED.SEARCH_BOX.PLACEHOLDER' | translate }}"></scholars-search-box>
+      <scholars-search-box 
+        aria-label="Search box" 
+        [styles]="getSearchStyles() | async" 
+        [view]="discoveryView | async" 
+        placeholder="{{ 'SHARED.SEARCH_BOX.PLACEHOLDER' | translate }}">
+      </scholars-search-box>
     </div>
   </div>
 </ng-template>
 
 <ng-template #homeStats>
   <div class="row justify-content-center home-stats">
     <div class="col-lg-6 col-md-8 col-sm-10 col-xs-12 d-flex flex-row justify-content-center">
-      <scholars-stats-box label="People" classifier="Person"></scholars-stats-box>
-      <scholars-stats-box label="Organizations" classifier="Organization"></scholars-stats-box>
-      <scholars-stats-box label="Research" classifier="Document"></scholars-stats-box>
+      <scholars-stats-box label="People" classifier="Person" aria-label="People statistics"></scholars-stats-box>
+      <scholars-stats-box label="Organizations" classifier="Organization" aria-label="Organizations statistics"></scholars-stats-box>
+      <scholars-stats-box label="Research" classifier="Document" aria-label="Research statistics"></scholars-stats-box>
```

Comment:
These are fine for now because there is no formal label.

However, with `label="People"` and label being defined inside the resulting expanded markup has `people` in it.
This can be changed to have an id and then given a `aria-labelledby` association.
This would avoid repeated the `aria-label`.